cylc/flow/option_parsers.py

```diff
@@ -345,6 +345,11 @@ class Options(Values):
     """
 
     def __init__(self, parser, overrides=None):
+        # Call parse_args() to add our "std options", for integration tests.
+        # TODO: std options should really be added in __init__() but that
+        # requires a bit of refactoring because option clashes are handled
+        # bass-ackwards ("overrides" are added before standard options).
+        parser.parse_args()
```

Comment:
@oliver-sanders - this is a bit ugly, but not the fault of this branch.

Reply:
Damn it, some functional tests broke; I thought this wouldn't have wider effects...

Reply:
(comment outdated)